apps/backend/src/applications/application.entity.ts

```diff
@@ -62,6 +63,23 @@ export class Application {
   @IsObject({ each: true })
   reviews: Review[];
 
+  toGetAllApplicationResponseDTO(): GetAllApplicationResponseDTO {
+    const meanRatingAllStages = 0; // TODO: calculate this
```

Comment:
💬 We did quickly talk about having `GetApplicationResponseDTO` extend `GetAllApplicationResponseDTO` but we can handle that in our PR that actually calculates the averages